src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  position: absolute;
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  padding: $padding;
+  max-width: $cell-size * 7 + $gap * 6 + $padding * 2;
+  gap: $gap;
+
+  &__day {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    border: 1px solid black;
+    background-color: #eee;
+    width: $cell-size;
+    height: $cell-size;
+    transition: all 0.5s ease-in-out;
```

Comment:
it's better to use directly the properties you changed on hover instead of "all"